Changed default cluster size back to 3

GoatCodeNL opened the pull request:
Signed-off-by: Niels van Esch <nvanesch@saa.nl>

#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Chart packaged and index updated (you can use `./release`)
- [x] CHANGELOG.md updated

<!--
Thank you for contributing to EventStore.Charts. Before you submit this PR we'd like to
make sure you are aware of our technical requirements and best practices:

* https://github.com/helm/charts/blob/master/CONTRIBUTING.md#technical-requirements
* https://github.com/helm/helm/tree/master/docs/chart_best_practices

For a quick overview across what we will look at reviewing your PR, please read
our review guidelines:

* https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md

Following our best practices right from the start will accelerate the review process and
help get your PR merged quicker.

When updates to your PR are requested, please add new commits and do not squash the
history. This will make it easier to identify new changes. The PR will be squashed
anyways when it is merged. Thanks.

For fast feedback, please @-mention maintainers that are listed in the Chart.yaml file.

Please make sure you test your changes before you push them. Once pushed,
Codefresh will run across your changes, lint and then install the chart.
Please check the results. We would like these checks to pass before we
even continue reviewing your changes.
-->

**What this PR does / why we need it**:
This PR restores the default clustersize to 3. to make it consistent with documentation, with other default values and expected behaviour.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes 36

**Special notes for your reviewer**:

riccardone commented:
I don't have permission to merge. This PR need to be merged

ameier38 commented:
We had changed the default cluster size back to 1 so the port forwarding worked out of the box. My mistake to not update the docs. Could we just update the docs to have the default value as 1?

riccardone commented:
can you please do that in a new PR and then we can eventually ask to close this one

GoatCodeNL commented:
Ofcourse I can change it. However I disagree that this would be the better default. 

* 3 is more like production
* 1 conflicts with current pdb default. This defaults to minAvailable of 2. It will never reach this. This means rolling updates get stuck and terminating instances do an unclean shutdown. 
* Portforwards are a debug situation. 

Honstly I would rather make a pr for you to add a note to the portforward docs. 

But I leave that up to you. Please let me know your preferred solution and I’ll make the PR.

ameier38 commented:
I agree that 3 is more like production, but also think that @hnicke raised a good point [here](https://github.com/EventStore/EventStore.Charts/pull/16#issuecomment-462126858) that the chart should work out of the box with port-forwarding so it is easy to get started. 

What if we added a `values-production.yml` similar to the [postgres chart](https://github.com/helm/charts/blob/master/stable/postgresql/values-production.yaml) and added `clusterSize: 3` there?

GoatCodeNL commented:
I'll see if I can create the PR somewhere this week. I'll create a PR with:

* updated documents
* changed PDB (defaulting to maxunavailable 1 in stead of minavailable 2
* values-production.yaml
* production advisory doc
